src/backend/aspen/workflows/nextstrain_run/export.py

```diff
@@ -60,111 +63,122 @@
 @click.command("save")
 @click.option("--phylo-run-id", type=int, required=True)
 @click.option("sequences_fh", "--sequences", type=click.File("w"), required=False)
-@click.option("selected_fh", "--selected", type=click.File("w"), required=False)
-@click.option("metadata_fh", "--metadata", type=click.File("w"), required=False)
+@click.option("selected_fh", "--selected", type=click.File("w"), required=True)
+@click.option("metadata_fh", "--metadata", type=click.File("w"), required=True)
 @click.option("builds_file_fh", "--builds-file", type=click.File("w"), required=True)
-@click.option(
-    "county_sequences_fh", "--county-sequences", type=click.File("w"), required=False
-)
-@click.option(
-    "county_metadata_fh", "--county-metadata", type=click.File("w"), required=False
-)
 @click.option("--test", type=bool, is_flag=True)
 def cli(
     phylo_run_id: int,
     sequences_fh: io.TextIOBase,
     selected_fh: io.TextIOBase,
     metadata_fh: io.TextIOBase,
     builds_file_fh: io.TextIOBase,
-    county_sequences_fh: io.TextIOBase,
-    county_metadata_fh: io.TextIOBase,
     test: bool,
 ):
     if test:
         print("Success!")
         return
+    aligned_gisaid_dump = export_run_config(
+        sequences_fh, selected_fh, metadata_fh, builds_file_fh
+    )
+    print(json.dumps(aligned_gisaid_dump))
+
+
+def export_run_config(
```

Comment:
Breaking most of this functionality out into a method that's easier to unit-test